lib/ceased_members_page.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+require_relative 'sobranie_page'
+require_relative 'ceased_member_row'
+
+class CeasedMembersPage < SobraniePage
+  field :members do
+    noko.xpath('//div[@class="row"]/p[a]').map do |p|
```

Comment:
I've opened and [issue](https://github.com/everypolitician-scrapers/macedonia-sobranie/issues/8) and have changed the PR description.